### ISSUE TITLE
Apotheosis Spawner Upgrades have incorrect recipe

### ISSUE THREAD

**Issue by kugamer** [EnigmaticaModpacks/Enigmatica6#4980]
### Modpack Version

1.3.0

### Describe your issue.

The redstone control recipe is incorrect, or at least didn't work. When I switched to a token of joy in offhand it worked. Currently JEI is showing token of sorrow. I did not check the others. 

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Singleplayer

**Comment by Quezler** [comment]
potential duplicate of #4952 (& #4950)

**Comment by atlgeek007** [comment]
Yeah this sounds like the confusion between "- Redstone Control" and "+ Redstone Control"

Maybe the display should not be "+" and "-" but should be "adds" and "removes" - but that would be up to Shadows to change in Apotheosis I think.
